src/App/pages/Profile/Profile.jsx

```diff
@@ -151,6 +171,13 @@ const Profile = () => {
         })}
       </PracticeList>
 
+      {currentTeacher.events && <CommingEvents>Ближайшие практики</CommingEvents> && (
+        <EventsList>
+          {new Date(events.startDate).getTime() > new Date().getTime() && (
```

Comment:
You have this condition twice, you can move it to helper

Reply:
```suggestion
          {new Date(events.startDate).getTime() > new Date().getTime() && (
           <>
          <CommingEvents>Ближайшие практики</CommingEvents>
          <EventsList>
            <PracticeCard hideMainInfo key={currentTeacher.id} {...currentTeacher} />
          </EventsList>
        </>
```

It is not that big and I will have to get current teacher twice, I think we should leave it like that